<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="radio_choose_model">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Model</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_choose_train_model">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_load_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selection</string>
   </property>
  </widget>
  <widget class="QFrame" name="model_frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_choose_model">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browse_model_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_model">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="train_frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>411</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_file">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>401</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>File Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Label</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_train">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Train MLP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>285</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Low pass</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_high_pass">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>315</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>265</y>
      <width>237</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FFt and filter (default 30MHZ -50MHZ)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>315</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>High Pass</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_low_pass">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>285</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>30000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_model_filename">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>random_file</string>
    </property>
   </widget>
   <widget class="QLabel" name="model_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Model filename</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="prediction_frame">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>30</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prediction File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_predict">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_predict_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_predict">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>137</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Prediction</string>
    </property>
   </widget>
  </widget>
  <zorder>model_frame</zorder>
  <zorder>radio_choose_model</zorder>
  <zorder>radio_choose_train_model</zorder>
  <zorder>label_load_2</zorder>
  <zorder>train_frame</zorder>
  <zorder>prediction_frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
